app/views/users/new.html.haml

```diff
@@ -19,21 +17,21 @@ Stripe.setPublishableKey("#{ENV['STRIPE_PUBLISHABLE_KEY']}");
           = f.text_field :full_name, label: 'Full Name'
           = hidden_field_tag :invitation_token, @invitation_token
         %fieldset.credit_card
-          %span.pay-ment-errors
+          %span.payment-errors
           .form-group
-            %label.control-label.col-sm-2 Credit Card Number
+            %label.control-label.col-sm-2(for="credit_card_number") Credit Card Number
             .col-sm-6
-              = text_field_tag :card_number, nil, "data-stripe" => "number", class: "form-control"
```

Comment:
Yes, I would not recommend Rails helpers here- good job switching to pure Haml/HTML.